packages/handlersjs-core/lib/handlers/typed-pipe-through.handler.ts

```diff
@@ -81,7 +81,7 @@ export class TypedPipeThroughHandler<A, B, C, D, E> extends Handler<A, E> {
 
       }
 
-      default: return of(undefined);
+      default: return of();
```

Comment:
Hm, didn't we noticed earlier that an empty `of()` blocks pipes?

Reply:
Yeah now that you mention it, in the proxy, `client-id-static-auth-request.handler.ts`, Abel had to return an Observable of an empty object to fix that.